### ISSUE TITLE
[Screen Reader - Microsoft Form Pro - Email] Links does not have discernible text

### ISSUE THREAD

**Issue by sonimishra** [OfficeDev/office-ui-fabric-react#11588]
User Experience:
User depending on screen reader might get affected if link does not have discernible text present on the screen and user will not be able to identify the control while navigating on screen.

Test Environment:
#URL:https://forms.office-int.com/
OS Version 1903 (OS Build 19018.1)
Tools: Accessibility insight tool
![MAS2 4 4_Link does not have text](https://user-images.githubusercontent.com/30456910/71655062-225cc380-2d5b-11ea-8840-eb919f284d76.png)

Note: User credentials should NOT be included in the bug.

Repro Steps:
1. Open URL: https://forms.office-int.com/ and login using valid credentials.
2. Navigate to any created survey and select it.
3. Navigate Send button present at the top and select it.
4. Navigate to Email tab and select it. Email screen will appear.
5. Verify the issue.
 
Actual Result:
Links does not have discernible text

Expected Result:
Links must have discernible text

MAS Reference:
MAS 2.4.4 - Link Purpose (In Context)
 
Reference Links
Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to EDAD team”.
Please reach out to C&AI Teams channel for any process related queries.

**Comment by sonimishra** [comment]
Product team comment: As the issue is with CKEditor , we are marking this bug as External.

**Comment by jdhuntington** [comment]
Hi @sonimishra - Thanks for the issue; unfortunately, this is not the repository behind the referenced product. Please comment if you disagree, otherwise I would like to close this issue.
